workers/unity/Assets/Fps/Tests/EditmodeTests/ChosenDeploymentAlphaLocatorFlowTests.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using Fps.Connection;
+using Improbable.Gdk.TestUtils.Editor;
+using Improbable.Gdk.Tools;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+
+namespace Fps.EditmodeTests
+{
+    [TestFixture]
+    public class ChosenDeploymentAlphaLocatorFlowTests
+    {
+        private SpatialdManager spatiald;
+        private Improbable.Worker.CInterop.Connection connection;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            if (!DevAuthTokenUtils.TryGenerate())
+            {
+                throw new ArgumentException("Could not generate a dev auth token.");
+            }
+
+            spatiald = SpatialdManager.Start().Result;
+        }
+
+        [OneTimeTearDown]
+        public void OneTimeTearDown()
+        {
+            spatiald?.Dispose();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            connection?.Dispose();
+            connection = null;
+        }
+
+        [Test]
+        public void CreateAsync_succeeds_if_there_is_matching_deployment()
+        {
+            using (var depl = spatiald.StartLocalDeployment("target_deployment", "default_launch.json").Result)
```

Comment:
Move the `default_launch.json` to the top to make replacement easier later

Reply:
👍